app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -259,6 +259,9 @@
         } else if (term.indexOf('quality:') >= 0 || term.indexOf('percentage:') >= 0) {
           filter = term.replace('quality:', '').replace('percentage:', '');
           addPredicate("quality", filter);
+        } else if (term.indexOf('stat:') >= 0) {
+          filter = term.split(':')[1];
+          addPredicate(filter, term.split(':')[2]);
```

Comment:
It will pass undefined, but the filter won't work until I type the entire search syntax (the same way typing "quality:" doesn't work). Maybe to avoid an annoying `console.error` I can check if all three terms exist...